mappers/linxoV2/accounts.ts

```diff
@@ -16,7 +16,8 @@ const getAccountStr = (account: AccountsEntity, accountName: string): string =>
   '',                               // payment_source
   '',                               // payment_destination
   account.iban,                     // iban
-  account.currency                  // currency
+  account.currency,                 // currency
+  account.bic                       // bic
```

Comment:
The BIC is not available in Linxo V2 file format:

```
ACCOUNT|account_uid|account_number|balance_date|balance|account_type|acceptable_uid_regexp|match_existing|owner|account_name|payment_source|payment_destination|iban|currency|OwnerAddress
|OwnerBirthDate|OwnerBirthPlace|OwnerIdentificationNumber|Usage|PhoneNumber|RegistrationDate|OwnerStatus|GroupingName
```

```suggestion
```

See the [documentation](https://developers.oxlin.io/reference-accounts-api/#section/File-structure)